she_codes_news/news/templates/news/createStory.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+
+{% if  request.user.is_authenticated %}
+<div class= "formbox">
+  
+    <form  id = "styledform" action="" method="post">
+        <h3> Create a New Story</h3>
+        {% csrf_token %}
+        {{ form.as_p }}
+        <input id = "form-submit" type="submit" value="submit">
+    </form>
+</div>
+{% else %}
+    <a href="{% url 'login' %}">Login</a>
+    <a href="{% url 'users:createAccount' %}"> Create Account</a>
```

Comment:
you could possibly have added this create account text to a template and just included it everywhere you needed it, since it seems to be repeated a little